### ISSUE TITLE
FileNotFoundError: [Errno 2] No such file or directory: 'unwrapped.nii'

### ISSUE THREAD

**Issue by LeeReid1** [QSMxT/QSMxT#107]
Thanks for putting this tool out there. 

I tried to run two datasets through qsmxt for QSM with default settings, and an error occured. The message was "FileNotFoundError: [Errno 2] No such file or directory: 'unwrapped.nii'", though I'm not familiar with the stacktrace and so not sure if this was truly the original issue or a result of romeo failing. There are a fair number of warnings. Please see below.

One dataset had 4 echos, another four. Both are 3D GRE. They were taken from the scanner as Real/Imag, converted to nifti with dxm2niix, converted into Phase/Mag niftis, renamed to match your BIDS examples, then provided to qsmxt. The Json files were simply the renamed jsons from the real/imag images, though I believe the only difference there would be the entry stating REAL/IMAGINARY/PHASE/MAGNITUDE, which I would guess is not read by your tool.

The raw images are poor quality - one acquisition was motion affected and the other was using near-default scanner settings - but I've unwrapped similar images previously with romeo, so an outright crash is unexpected. I've put nifti/json info below too.

Thanks!

OS: MacOS
Chipset: ARM
Version: QSMxT v6.3.2
Installation: Docker image
Scanner manufacturer: GE
Sequence: SWAN

<details>
  <summary>Click me for the stack trace</summary>
Stack trace:

```
(base) root@3cf99f574211:/neurodesktop-storage/somepatient# qsmxt /neurodesktop-storage/somepatient/bids /neurodesktop-storage/somepatient/result

=== Desired outputs ===
 qsm: Quantitative Susceptibility Mapping (QSM)
 swi: Susceptibility Weighted Imaging (SWI)
 t2s: T2* maps
 r2s: R2* maps
 seg: Segmentations (requires qsm)
 analysis: QSM across segmented ROIs (requires qsm+seg)
 template: GRE group space + GRE/QSM templates (requires qsm)

Enter desired images (space-separated) [default - qsm]: qsm

=== Premade QSM pipelines ===
default: Default QSMxT settings (GRE; assumes human brain)
gre: Applies suggested settings for 3D-GRE images
epi: Applies suggested settings for 3D-EPI images (assumes human brain)
bet: Applies a traditional BET-masking approach (artefact reduction unavailable; assumes human brain)
fast: Applies a set of fast algorithms
body: Applies suggested settings for non-brain applications
nextqsm: Applies suggested settings for running the NeXtQSM algorithm (assumes human brain)

Select a premade to begin [default - 'default']: default

=== QSMxT - Settings Menu ===

(1) Desired outputs:
 - Quantitative Susceptibility Mapping (QSM): Yes
 - Susceptibility Weighted Imaging (SWI): No
 - T2* mapping: No
 - R2* mapping: No
 - Segmentations: No
 - Analysis CSVs: No
 - GRE/QSM template space: No

(2) QSM pipeline: default

(3) [ADVANCED] QSM masking:
 - Use existing masks if available: No
 - Masking algorithm: threshold (phase-based)
   - Two-pass artefact reduction: Enabled
   - Threshold algorithm: otsu (x1.5 for single-pass; x1.3 for two-pass)
   - Hole-filling algorithm: morphological+gaussian
   - Erosions: 2 erosions for single-pass; 0 erosions for two-pass

(4) [ADVANCED] QSM phase processing:
 - Axial resampling: Enabled (obliquity threshold = 10)
 - Multi-echo combination: B0 mapping (using ROMEO)
 - Phase unwrapping: romeo
 - Background field removal: pdf
 - Dipole inversion: rts
 - Referencing: mean

Guidelines compliant! (see https://arxiv.org/abs/2307.02306)

Run command: qsmxt bids result --auto_yes

Enter a number to customize; enter 'run' to run: run
[INFO]: QSMxT v6.3.2
[INFO]: Python interpreter: /opt/miniconda-4.7.12.1/bin/python3.8
[INFO]: Command: qsmxt /neurodesktop-storage/somepatient/bids /neurodesktop-storage/somepatient/result --labels_file '/opt/miniconda-4.7.12.1/lib/python3.8/site-packages/qsmxt/aseg_labels.csv' --auto_yes
[INFO]: Guidelines compliant! (see https://arxiv.org/abs/2307.02306)
[INFO]: Creating QSMxT workflow for sub-1...
[INFO]: Running using MultiProc plugin with n_procs=10
240121-20:05:10,749 nipype.workflow INFO:
	 Workflow workflow settings: ['check', 'execution', 'logging', 'monitoring']
240121-20:05:10,775 nipype.workflow INFO:
	 Running in parallel.
240121-20:05:10,782 nipype.workflow INFO:
	 [MultiProc] Running 0 tasks, and 4 jobs ready. Free memory (GB): 17.56/17.56, Free processors: 10/10.
240121-20:05:10,873 nipype.workflow INFO:
	 [Node] Setting-up "workflow.sub-1.sub-1.qsmxt.func_read-json-se" in "/neurodesktop-storage/somepatient/result/workflow/sub-1/sub-1/qsmxt/func_read-json-se".
240121-20:05:10,875 nipype.workflow INFO:
	 [Node] Setting-up "workflow.sub-1.sub-1.qsmxt.nibabel_as-canonical" in "/neurodesktop-storage/somepatient/result/workflow/sub-1/sub-1/qsmxt/nibabel_as-canonical".
240121-20:05:10,888 nipype.workflow INFO:
	 [Node] Executing "func_read-json-se" <nipype.interfaces.utility.wrappers.Function>
240121-20:05:10,875 nipype.workflow INFO:
	 [Node] Setting-up "workflow.sub-1.sub-1.qsmxt.nibabel_read-nii" in "/neurodesktop-storage/somepatient/result/workflow/sub-1/sub-1/qsmxt/nibabel_read-nii".
240121-20:05:10,934 nipype.workflow INFO:
	 [Node] Finished "func_read-json-se", elapsed time 0.0037s.
240121-20:05:10,938 nipype.workflow INFO:
	 [Node] Executing "nibabel_read-nii" <nipype.interfaces.utility.wrappers.Function>
240121-20:05:10,889 nipype.workflow INFO:
	 [Node] Executing "nibabel_as-canonical" <nipype.interfaces.utility.wrappers.Function>
240121-20:05:10,979 nipype.workflow INFO:
	 [Node] Finished "nibabel_read-nii", elapsed time 0.005502s.
240121-20:05:11,31 nipype.workflow INFO:
	 [Node] Finished "nibabel_as-canonical", elapsed time 0.057737s.
240121-20:05:12,791 nipype.workflow INFO:
	 [Job 1] Completed (workflow.sub-1.sub-1.qsmxt.func_read-json-se).
240121-20:05:12,799 nipype.workflow INFO:
	 [Job 2] Completed (workflow.sub-1.sub-1.qsmxt.nibabel_read-nii).
240121-20:05:12,804 nipype.workflow INFO:
	 [Job 3] Completed (workflow.sub-1.sub-1.qsmxt.nibabel_as-canonical).
240121-20:05:12,810 nipype.workflow INFO:
	 [MultiProc] Running 0 tasks, and 5 jobs ready. Free memory (GB): 17.56/17.56, Free processors: 10/10.
240121-20:05:12,915 nipype.workflow INFO:
	 [Node] Setting-up "_func_read-json-me0" in "/neurodesktop-storage/somepatient/result/workflow/sub-1/sub-1/qsmxt/func_read-json-me/mapflow/_func_read-json-me0".
240121-20:05:12,917 nipype.workflow INFO:
	 [Node] Setting-up "_func_read-json-me2" in "/neurodesktop-storage/somepatient/result/workflow/sub-1/sub-1/qsmxt/func_read-json-me/mapflow/_func_read-json-me2".
240121-20:05:12,928 nipype.workflow INFO:
	 [Node] Executing "_func_read-json-me0" <nipype.interfaces.utility.wrappers.Function>
240121-20:05:12,929 nipype.workflow INFO:
	 [Node] Executing "_func_read-json-me2" <nipype.interfaces.utility.wrappers.Function>
240121-20:05:12,968 nipype.workflow INFO:
	 [Node] Finished "_func_read-json-me0", elapsed time 0.00387s.
240121-20:05:12,918 nipype.workflow INFO:
	 [Node] Setting-up "_func_read-json-me3" in "/neurodesktop-storage/somepatient/result/workflow/sub-1/sub-1/qsmxt/func_read-json-me/mapflow/_func_read-json-me3".
240121-20:05:12,983 nipype.workflow INFO:
	 [Node] Executing "_func_read-json-me3" <nipype.interfaces.utility.wrappers.Function>
240121-20:05:12,916 nipype.workflow INFO:
	 [Node] Setting-up "_func_read-json-me1" in "/neurodesktop-storage/somepatient/result/workflow/sub-1/sub-1/qsmxt/func_read-json-me/mapflow/_func_read-json-me1".
240121-20:05:13,22 nipype.workflow INFO:
	 [Node] Finished "_func_read-json-me3", elapsed time 0.003373s.
240121-20:05:13,31 nipype.workflow INFO:
	 [Node] Executing "_func_read-json-me1" <nipype.interfaces.utility.wrappers.Function>
240121-20:05:13,67 nipype.workflow INFO:
	 [Node] Finished "_func_read-json-me1", elapsed time 0.002244s.
240121-20:05:12,968 nipype.workflow INFO:
	 [Node] Finished "_func_read-json-me2", elapsed time 0.003941s.
240121-20:05:14,791 nipype.workflow INFO:
	 [Job 28] Completed (_func_read-json-me0).
240121-20:05:14,796 nipype.workflow INFO:
	 [Job 29] Completed (_func_read-json-me1).
240121-20:05:14,801 nipype.workflow INFO:
	 [Job 30] Completed (_func_read-json-me2).
240121-20:05:14,805 nipype.workflow INFO:
	 [Job 31] Completed (_func_read-json-me3).
240121-20:05:14,811 nipype.workflow INFO:
	 [MultiProc] Running 0 tasks, and 5 jobs ready. Free memory (GB): 17.56/17.56, Free processors: 10/10.
240121-20:05:14,873 nipype.workflow INFO:
	 [Node] Setting-up "_nibabel_numpy_scale-phase0" in "/neurodesktop-storage/somepatient/result/workflow/sub-1/sub-1/qsmxt/nibabel_numpy_scale-phase/mapflow/_nibabel_numpy_scale-phase0".
240121-20:05:14,876 nipype.workflow INFO:
	 [Node] Setting-up "_nibabel_numpy_scale-phase2" in "/neurodesktop-storage/somepatient/result/workflow/sub-1/sub-1/qsmxt/nibabel_numpy_scale-phase/mapflow/_nibabel_numpy_scale-phase2".
240121-20:05:14,877 nipype.workflow INFO:
	 [Node] Setting-up "_nibabel_numpy_scale-phase3" in "/neurodesktop-storage/somepatient/result/workflow/sub-1/sub-1/qsmxt/nibabel_numpy_scale-phase/mapflow/_nibabel_numpy_scale-phase3".
240121-20:05:14,886 nipype.workflow INFO:
	 [Node] Executing "_nibabel_numpy_scale-phase2" <qsmxt.interfaces.nipype_interface_processphase.ScalePhaseInterface>
240121-20:05:14,889 nipype.workflow INFO:
	 [Node] Executing "_nibabel_numpy_scale-phase0" <qsmxt.interfaces.nipype_interface_processphase.ScalePhaseInterface>
240121-20:05:14,892 nipype.workflow INFO:
	 [Node] Setting-up "_func_read-json-me0" in "/neurodesktop-storage/somepatient/result/workflow/sub-1/sub-1/qsmxt/func_read-json-me/mapflow/_func_read-json-me0".
240121-20:05:14,898 nipype.workflow INFO:
	 [Node] Cached "_func_read-json-me0" - collecting precomputed outputs
240121-20:05:14,900 nipype.workflow INFO:
	 [Node] "_func_read-json-me0" found cached.
240121-20:05:14,902 nipype.workflow INFO:
	 [Node] Setting-up "_func_read-json-me1" in "/neurodesktop-storage/somepatient/result/workflow/sub-1/sub-1/qsmxt/func_read-json-me/mapflow/_func_read-json-me1".
240121-20:05:14,905 nipype.workflow INFO:
	 [Node] Cached "_func_read-json-me1" - collecting precomputed outputs
240121-20:05:14,906 nipype.workflow INFO:
	 [Node] "_func_read-json-me1" found cached.
240121-20:05:14,907 nipype.workflow INFO:
	 [Node] Setting-up "_func_read-json-me2" in "/neurodesktop-storage/somepatient/result/workflow/sub-1/sub-1/qsmxt/func_read-json-me/mapflow/_func_read-json-me2".
240121-20:05:14,909 nipype.workflow INFO:
	 [Node] Cached "_func_read-json-me2" - collecting precomputed outputs
240121-20:05:14,910 nipype.workflow INFO:
	 [Node] "_func_read-json-me2" found cached.
240121-20:05:14,911 nipype.workflow INFO:
	 [Node] Setting-up "_func_read-json-me3" in "/neurodesktop-storage/somepatient/result/workflow/sub-1/sub-1/qsmxt/func_read-json-me/mapflow/_func_read-json-me3".
240121-20:05:14,913 nipype.workflow INFO:
	 [Node] Cached "_func_read-json-me3" - collecting precomputed outputs
240121-20:05:14,915 nipype.workflow INFO:
	 [Node] "_func_read-json-me3" found cached.
240121-20:05:14,877 nipype.workflow INFO:
	 [Node] Setting-up "_nibabel_numpy_scale-phase1" in "/neurodesktop-storage/somepatient/result/workflow/sub-1/sub-1/qsmxt/nibabel_numpy_scale-phase/mapflow/_nibabel_numpy_scale-phase1".
240121-20:05:14,886 nipype.workflow INFO:
	 [Node] Executing "_nibabel_numpy_scale-phase3" <qsmxt.interfaces.nipype_interface_processphase.ScalePhaseInterface>
240121-20:05:14,940 nipype.workflow INFO:
	 [Node] Executing "_nibabel_numpy_scale-phase1" <qsmxt.interfaces.nipype_interface_processphase.ScalePhaseInterface>
240121-20:05:16,791 nipype.workflow INFO:
	 [Job 0] Completed (workflow.sub-1.sub-1.qsmxt.func_read-json-me).
240121-20:05:16,798 nipype.workflow INFO:
	 [MultiProc] Running 4 tasks, and 0 jobs ready. Free memory (GB): 16.76/17.56, Free processors: 6/10.
                     Currently running:
                       * _nibabel_numpy_scale-phase3
                       * _nibabel_numpy_scale-phase2
                       * _nibabel_numpy_scale-phase1
                       * _nibabel_numpy_scale-phase0
240121-20:05:23,990 nipype.workflow INFO:
	 [Node] Finished "_nibabel_numpy_scale-phase2", elapsed time 9.101498s.
240121-20:05:24,77 nipype.workflow INFO:
	 [Node] Finished "_nibabel_numpy_scale-phase3", elapsed time 9.138709s.
240121-20:05:24,82 nipype.workflow INFO:
	 [Node] Finished "_nibabel_numpy_scale-phase1", elapsed time 9.10258s.
240121-20:05:24,84 nipype.workflow INFO:
	 [Node] Finished "_nibabel_numpy_scale-phase0", elapsed time 9.157892s.
240121-20:05:24,818 nipype.workflow INFO:
	 [Job 32] Completed (_nibabel_numpy_scale-phase0).
240121-20:05:24,825 nipype.workflow INFO:
	 [Job 33] Completed (_nibabel_numpy_scale-phase1).
240121-20:05:24,831 nipype.workflow INFO:
	 [Job 34] Completed (_nibabel_numpy_scale-phase2).
240121-20:05:24,837 nipype.workflow INFO:
	 [Job 35] Completed (_nibabel_numpy_scale-phase3).
240121-20:05:24,843 nipype.workflow INFO:
	 [MultiProc] Running 0 tasks, and 1 jobs ready. Free memory (GB): 17.56/17.56, Free processors: 10/10.
240121-20:05:24,909 nipype.workflow INFO:
	 [Node] Setting-up "_nibabel_numpy_scale-phase0" in "/neurodesktop-storage/somepatient/result/workflow/sub-1/sub-1/qsmxt/nibabel_numpy_scale-phase/mapflow/_nibabel_numpy_scale-phase0".
240121-20:05:24,913 nipype.workflow INFO:
	 [Node] Cached "_nibabel_numpy_scale-phase0" - collecting precomputed outputs
240121-20:05:24,914 nipype.workflow INFO:
	 [Node] "_nibabel_numpy_scale-phase0" found cached.
240121-20:05:24,915 nipype.workflow INFO:
	 [Node] Setting-up "_nibabel_numpy_scale-phase1" in "/neurodesktop-storage/somepatient/result/workflow/sub-1/sub-1/qsmxt/nibabel_numpy_scale-phase/mapflow/_nibabel_numpy_scale-phase1".
240121-20:05:24,917 nipype.workflow INFO:
	 [Node] Cached "_nibabel_numpy_scale-phase1" - collecting precomputed outputs
240121-20:05:24,918 nipype.workflow INFO:
	 [Node] "_nibabel_numpy_scale-phase1" found cached.
240121-20:05:24,920 nipype.workflow INFO:
	 [Node] Setting-up "_nibabel_numpy_scale-phase2" in "/neurodesktop-storage/somepatient/result/workflow/sub-1/sub-1/qsmxt/nibabel_numpy_scale-phase/mapflow/_nibabel_numpy_scale-phase2".
240121-20:05:24,922 nipype.workflow INFO:
	 [Node] Cached "_nibabel_numpy_scale-phase2" - collecting precomputed outputs
240121-20:05:24,923 nipype.workflow INFO:
	 [Node] "_nibabel_numpy_scale-phase2" found cached.
240121-20:05:24,924 nipype.workflow INFO:
	 [Node] Setting-up "_nibabel_numpy_scale-phase3" in "/neurodesktop-storage/somepatient/result/workflow/sub-1/sub-1/qsmxt/nibabel_numpy_scale-phase/mapflow/_nibabel_numpy_scale-phase3".
240121-20:05:24,925 nipype.workflow INFO:
	 [Node] Cached "_nibabel_numpy_scale-phase3" - collecting precomputed outputs
240121-20:05:24,926 nipype.workflow INFO:
	 [Node] "_nibabel_numpy_scale-phase3" found cached.
240121-20:05:26,825 nipype.workflow INFO:
	 [Job 4] Completed (workflow.sub-1.sub-1.qsmxt.nibabel_numpy_scale-phase).
240121-20:05:26,837 nipype.workflow INFO:
	 [MultiProc] Running 0 tasks, and 1 jobs ready. Free memory (GB): 17.56/17.56, Free processors: 10/10.
240121-20:05:28,832 nipype.workflow INFO:
	 [MultiProc] Running 0 tasks, and 4 jobs ready. Free memory (GB): 17.56/17.56, Free processors: 10/10.
240121-20:05:28,905 nipype.workflow INFO:
	 [Node] Setting-up "_nibabel_numpy_nilearn_axial-resampling0" in "/neurodesktop-storage/somepatient/result/workflow/sub-1/sub-1/qsmxt/nibabel_numpy_nilearn_axial-resampling/mapflow/_nibabel_numpy_nilearn_axial-resampling0".
240121-20:05:28,906 nipype.workflow INFO:
	 [Node] Setting-up "_nibabel_numpy_nilearn_axial-resampling1" in "/neurodesktop-storage/somepatient/result/workflow/sub-1/sub-1/qsmxt/nibabel_numpy_nilearn_axial-resampling/mapflow/_nibabel_numpy_nilearn_axial-resampling1".
240121-20:05:28,908 nipype.workflow INFO:
	 [Node] Setting-up "_nibabel_numpy_nilearn_axial-resampling2" in "/neurodesktop-storage/somepatient/result/workflow/sub-1/sub-1/qsmxt/nibabel_numpy_nilearn_axial-resampling/mapflow/_nibabel_numpy_nilearn_axial-resampling2".
240121-20:05:28,909 nipype.workflow INFO:
	 [Node] Setting-up "_nibabel_numpy_nilearn_axial-resampling3" in "/neurodesktop-storage/somepatient/result/workflow/sub-1/sub-1/qsmxt/nibabel_numpy_nilearn_axial-resampling/mapflow/_nibabel_numpy_nilearn_axial-resampling3".
240121-20:05:28,915 nipype.workflow INFO:
	 [Node] Executing "_nibabel_numpy_nilearn_axial-resampling0" <qsmxt.interfaces.nipype_interface_axialsampling.AxialSamplingInterface>
240121-20:05:28,916 nipype.workflow INFO:
	 [Node] Executing "_nibabel_numpy_nilearn_axial-resampling1" <qsmxt.interfaces.nipype_interface_axialsampling.AxialSamplingInterface>
240121-20:05:28,918 nipype.workflow INFO:
	 [Node] Executing "_nibabel_numpy_nilearn_axial-resampling2" <qsmxt.interfaces.nipype_interface_axialsampling.AxialSamplingInterface>
240121-20:05:28,919 nipype.workflow INFO:
	 [Node] Executing "_nibabel_numpy_nilearn_axial-resampling3" <qsmxt.interfaces.nipype_interface_axialsampling.AxialSamplingInterface>
240121-20:05:30,827 nipype.workflow INFO:
	 [MultiProc] Running 4 tasks, and 0 jobs ready. Free memory (GB): 1.56/17.56, Free processors: 6/10.
                     Currently running:
                       * _nibabel_numpy_nilearn_axial-resampling3
                       * _nibabel_numpy_nilearn_axial-resampling2
                       * _nibabel_numpy_nilearn_axial-resampling1
                       * _nibabel_numpy_nilearn_axial-resampling0
240121-20:06:37,32 nipype.workflow INFO:
	 [Node] Finished "_nibabel_numpy_nilearn_axial-resampling3", elapsed time 68.107222s.
240121-20:06:37,880 nipype.workflow INFO:
	 [Node] Finished "_nibabel_numpy_nilearn_axial-resampling2", elapsed time 68.959046s.
240121-20:06:39,12 nipype.workflow INFO:
	 [Job 38] Completed (_nibabel_numpy_nilearn_axial-resampling2).
240121-20:06:39,18 nipype.workflow INFO:
	 [Job 39] Completed (_nibabel_numpy_nilearn_axial-resampling3).
240121-20:06:39,20 nipype.workflow INFO:
	 [MultiProc] Running 2 tasks, and 0 jobs ready. Free memory (GB): 9.56/17.56, Free processors: 8/10.
                     Currently running:
                       * _nibabel_numpy_nilearn_axial-resampling1
                       * _nibabel_numpy_nilearn_axial-resampling0
240121-20:06:39,37 nipype.workflow INFO:
	 [Node] Finished "_nibabel_numpy_nilearn_axial-resampling1", elapsed time 70.117193s.
240121-20:06:39,268 nipype.workflow INFO:
	 [Node] Finished "_nibabel_numpy_nilearn_axial-resampling0", elapsed time 70.349394s.
240121-20:06:41,21 nipype.workflow INFO:
	 [Job 36] Completed (_nibabel_numpy_nilearn_axial-resampling0).
240121-20:06:41,27 nipype.workflow INFO:
	 [Job 37] Completed (_nibabel_numpy_nilearn_axial-resampling1).
240121-20:06:41,33 nipype.workflow INFO:
	 [MultiProc] Running 0 tasks, and 1 jobs ready. Free memory (GB): 17.56/17.56, Free processors: 10/10.
240121-20:06:41,116 nipype.workflow INFO:
	 [Node] Setting-up "_nibabel_numpy_nilearn_axial-resampling0" in "/neurodesktop-storage/somepatient/result/workflow/sub-1/sub-1/qsmxt/nibabel_numpy_nilearn_axial-resampling/mapflow/_nibabel_numpy_nilearn_axial-resampling0".
240121-20:06:41,119 nipype.workflow INFO:
	 [Node] Cached "_nibabel_numpy_nilearn_axial-resampling0" - collecting precomputed outputs
240121-20:06:41,120 nipype.workflow INFO:
	 [Node] "_nibabel_numpy_nilearn_axial-resampling0" found cached.
240121-20:06:41,122 nipype.workflow INFO:
	 [Node] Setting-up "_nibabel_numpy_nilearn_axial-resampling1" in "/neurodesktop-storage/somepatient/result/workflow/sub-1/sub-1/qsmxt/nibabel_numpy_nilearn_axial-resampling/mapflow/_nibabel_numpy_nilearn_axial-resampling1".
240121-20:06:41,125 nipype.workflow INFO:
	 [Node] Cached "_nibabel_numpy_nilearn_axial-resampling1" - collecting precomputed outputs
240121-20:06:41,126 nipype.workflow INFO:
	 [Node] "_nibabel_numpy_nilearn_axial-resampling1" found cached.
240121-20:06:41,128 nipype.workflow INFO:
	 [Node] Setting-up "_nibabel_numpy_nilearn_axial-resampling2" in "/neurodesktop-storage/somepatient/result/workflow/sub-1/sub-1/qsmxt/nibabel_numpy_nilearn_axial-resampling/mapflow/_nibabel_numpy_nilearn_axial-resampling2".
240121-20:06:41,129 nipype.workflow INFO:
	 [Node] Cached "_nibabel_numpy_nilearn_axial-resampling2" - collecting precomputed outputs
240121-20:06:41,130 nipype.workflow INFO:
	 [Node] "_nibabel_numpy_nilearn_axial-resampling2" found cached.
240121-20:06:41,131 nipype.workflow INFO:
	 [Node] Setting-up "_nibabel_numpy_nilearn_axial-resampling3" in "/neurodesktop-storage/somepatient/result/workflow/sub-1/sub-1/qsmxt/nibabel_numpy_nilearn_axial-resampling/mapflow/_nibabel_numpy_nilearn_axial-resampling3".
240121-20:06:41,133 nipype.workflow INFO:
	 [Node] Cached "_nibabel_numpy_nilearn_axial-resampling3" - collecting precomputed outputs
240121-20:06:41,134 nipype.workflow INFO:
	 [Node] "_nibabel_numpy_nilearn_axial-resampling3" found cached.
240121-20:06:43,23 nipype.workflow INFO:
	 [Job 5] Completed (workflow.sub-1.sub-1.qsmxt.nibabel_numpy_nilearn_axial-resampling).
240121-20:06:43,33 nipype.workflow INFO:
	 [MultiProc] Running 0 tasks, and 5 jobs ready. Free memory (GB): 17.56/17.56, Free processors: 10/10.
240121-20:06:43,90 nipype.workflow INFO:
	 [Node] Setting-up "workflow.sub-1.sub-1.qsmxt.mask_workflow.nibabal-numpy_combine-magnitude" in "/neurodesktop-storage/somepatient/result/workflow/sub-1/sub-1/qsmxt/mask_workflow/nibabal-numpy_combine-magnitude".
240121-20:06:43,94 nipype.workflow INFO:
	 [Node] Setting-up "workflow.sub-1.sub-1.qsmxt.mask-intermediate_workflow.func_get-first-phase" in "/neurodesktop-storage/somepatient/result/workflow/sub-1/sub-1/qsmxt/mask-intermediate_workflow/func_get-first-phase".
240121-20:06:43,126 nipype.workflow INFO:
	 [Node] Executing "nibabal-numpy_combine-magnitude" <qsmxt.interfaces.nipype_interface_combinemagnitude.CombineMagnitudeInterface>
240121-20:06:43,127 nipype.workflow INFO:
	 [Node] Executing "func_get-first-phase" <nipype.interfaces.utility.wrappers.Function>
240121-20:06:43,132 nipype.workflow INFO:
	 [Node] Finished "func_get-first-phase", elapsed time 0.002313s.
240121-20:06:43,91 nipype.workflow INFO:
	 [Node] Setting-up "workflow.sub-1.sub-1.qsmxt.mask_workflow.func_get-first-phase" in "/neurodesktop-storage/somepatient/result/workflow/sub-1/sub-1/qsmxt/mask_workflow/func_get-first-phase".
240121-20:06:43,158 nipype.workflow INFO:
	 [Node] Executing "func_get-first-phase" <nipype.interfaces.utility.wrappers.Function>
240121-20:06:43,162 nipype.workflow INFO:
	 [Node] Finished "func_get-first-phase", elapsed time 0.001636s.
240121-20:06:43,91 nipype.workflow INFO:
	 [Node] Setting-up "workflow.sub-1.sub-1.qsmxt.mrt_romeo_combine" in "/neurodesktop-storage/somepatient/result/workflow/sub-1/sub-1/qsmxt/mrt_romeo_combine".
240121-20:06:43,210 nipype.workflow INFO:
	 [Node] Executing "mrt_romeo_combine" <qsmxt.interfaces.nipype_interface_romeo.RomeoB0Interface>
240121-20:06:43,92 nipype.workflow INFO:
	 [Node] Setting-up "workflow.sub-1.sub-1.qsmxt.mask-intermediate_workflow.nibabal-numpy_combine-magnitude" in "/neurodesktop-storage/somepatient/result/workflow/sub-1/sub-1/qsmxt/mask-intermediate_workflow/nibabal-numpy_combine-magnitude".
240121-20:06:43,260 nipype.workflow INFO:
	 [Node] Executing "nibabal-numpy_combine-magnitude" <qsmxt.interfaces.nipype_interface_combinemagnitude.CombineMagnitudeInterface>
240121-20:06:45,27 nipype.workflow INFO:
	 [Job 8] Completed (workflow.sub-1.sub-1.qsmxt.mask_workflow.func_get-first-phase).
240121-20:06:45,30 nipype.workflow INFO:
	 [Job 10] Completed (workflow.sub-1.sub-1.qsmxt.mask-intermediate_workflow.func_get-first-phase).
240121-20:06:45,32 nipype.workflow INFO:
	 [MultiProc] Running 3 tasks, and 0 jobs ready. Free memory (GB): 9.16/17.56, Free processors: 9/10.
                     Currently running:
                       * workflow.sub-1.sub-1.qsmxt.mask-intermediate_workflow.nibabal-numpy_combine-magnitude
                       * workflow.sub-1.sub-1.qsmxt.mask_workflow.nibabal-numpy_combine-magnitude
                       * workflow.sub-1.sub-1.qsmxt.mrt_romeo_combine
240121-20:06:56,601 nipype.workflow INFO:
	 [Node] Finished "nibabal-numpy_combine-magnitude", elapsed time 13.471443s.
240121-20:06:56,733 nipype.workflow INFO:
	 [Node] Finished "nibabal-numpy_combine-magnitude", elapsed time 13.468879s.
240121-20:06:57,62 nipype.workflow INFO:
	 [Job 7] Completed (workflow.sub-1.sub-1.qsmxt.mask_workflow.nibabal-numpy_combine-magnitude).
240121-20:06:57,65 nipype.workflow INFO:
	 [Job 9] Completed (workflow.sub-1.sub-1.qsmxt.mask-intermediate_workflow.nibabal-numpy_combine-magnitude).
240121-20:06:57,66 nipype.workflow INFO:
	 [MultiProc] Running 1 tasks, and 1 jobs ready. Free memory (GB): 9.56/17.56, Free processors: 11/10.
                     Currently running:
                       * workflow.sub-1.sub-1.qsmxt.mrt_romeo_combine
240121-20:06:57,98 nipype.workflow INFO:
	 [Node] Setting-up "workflow.sub-1.sub-1.qsmxt.mask_workflow.romeo-voxelquality" in "/neurodesktop-storage/somepatient/result/workflow/sub-1/sub-1/qsmxt/mask_workflow/romeo-voxelquality".
240121-20:06:57,112 nipype.workflow INFO:
	 [Node] Executing "romeo-voxelquality" <qsmxt.interfaces.nipype_interface_phaseweights.RomeoMaskingInterface>
240121-20:06:59,66 nipype.workflow INFO:
	 [MultiProc] Running 2 tasks, and 0 jobs ready. Free memory (GB): 6.56/17.56, Free processors: 12/10.
                     Currently running:
                       * workflow.sub-1.sub-1.qsmxt.mask_workflow.romeo-voxelquality
                       * workflow.sub-1.sub-1.qsmxt.mrt_romeo_combine
240121-20:10:48,236 nipype.workflow INFO:
	 [Node] Finished "romeo-voxelquality", elapsed time 231.121685s.
240121-20:10:49,442 nipype.workflow INFO:
	 [Job 13] Completed (workflow.sub-1.sub-1.qsmxt.mask_workflow.romeo-voxelquality).
240121-20:10:49,446 nipype.workflow INFO:
	 [MultiProc] Running 1 tasks, and 2 jobs ready. Free memory (GB): 9.56/17.56, Free processors: 11/10.
                     Currently running:
                       * workflow.sub-1.sub-1.qsmxt.mrt_romeo_combine
240121-20:10:49,501 nipype.workflow INFO:
	 [Node] Setting-up "workflow.sub-1.sub-1.qsmxt.mask_workflow.scipy_numpy_nibabel_threshold-masking" in "/neurodesktop-storage/somepatient/result/workflow/sub-1/sub-1/qsmxt/mask_workflow/scipy_numpy_nibabel_threshold-masking".
240121-20:10:49,528 nipype.workflow INFO:
	 [Node] Executing "scipy_numpy_nibabel_threshold-masking" <qsmxt.interfaces.nipype_interface_masking.MaskingInterface>
240121-20:10:49,502 nipype.workflow INFO:
	 [Node] Setting-up "workflow.sub-1.sub-1.qsmxt.mask-intermediate_workflow.scipy_numpy_nibabel_threshold-masking" in "/neurodesktop-storage/somepatient/result/workflow/sub-1/sub-1/qsmxt/mask-intermediate_workflow/scipy_numpy_nibabel_threshold-masking".
240121-20:10:49,577 nipype.workflow INFO:
	 [Node] Executing "scipy_numpy_nibabel_threshold-masking" <qsmxt.interfaces.nipype_interface_masking.MaskingInterface>
240121-20:10:51,451 nipype.workflow INFO:
	 [MultiProc] Running 3 tasks, and 0 jobs ready. Free memory (GB): 9.16/17.56, Free processors: 9/10.
                     Currently running:
                       * workflow.sub-1.sub-1.qsmxt.mask-intermediate_workflow.scipy_numpy_nibabel_threshold-masking
                       * workflow.sub-1.sub-1.qsmxt.mask_workflow.scipy_numpy_nibabel_threshold-masking
                       * workflow.sub-1.sub-1.qsmxt.mrt_romeo_combine
240121-20:10:55,500 nipype.workflow INFO:
	 [Node] Finished "scipy_numpy_nibabel_threshold-masking", elapsed time 5.919023s.
240121-20:10:57,461 nipype.workflow INFO:
	 [Job 15] Completed (workflow.sub-1.sub-1.qsmxt.mask-intermediate_workflow.scipy_numpy_nibabel_threshold-masking).
240121-20:10:57,465 nipype.workflow INFO:
	 [MultiProc] Running 2 tasks, and 0 jobs ready. Free memory (GB): 9.36/17.56, Free processors: 10/10.
                     Currently running:
                       * workflow.sub-1.sub-1.qsmxt.mask_workflow.scipy_numpy_nibabel_threshold-masking
                       * workflow.sub-1.sub-1.qsmxt.mrt_romeo_combine
240121-20:10:59,384 nipype.workflow INFO:
	 [Node] Finished "scipy_numpy_nibabel_threshold-masking", elapsed time 9.841126s.
240121-20:10:59,463 nipype.workflow INFO:
	 [Job 14] Completed (workflow.sub-1.sub-1.qsmxt.mask_workflow.scipy_numpy_nibabel_threshold-masking).
240121-20:10:59,465 nipype.workflow INFO:
	 [MultiProc] Running 1 tasks, and 0 jobs ready. Free memory (GB): 9.56/17.56, Free processors: 11/10.
                     Currently running:
                       * workflow.sub-1.sub-1.qsmxt.mrt_romeo_combine
240121-20:11:15,933 nipype.workflow INFO:
	 [Node] Finished "mrt_romeo_combine", elapsed time 272.714437s.
240121-20:11:15,935 nipype.workflow WARNING:
	 Storing result file without outputs
240121-20:11:15,940 nipype.workflow WARNING:
	 [Node] Error on "workflow.sub-1.sub-1.qsmxt.mrt_romeo_combine" (/neurodesktop-storage/somepatient/result/workflow/sub-1/sub-1/qsmxt/mrt_romeo_combine)
240121-20:11:17,526 nipype.workflow ERROR:
	 Node mrt_romeo_combine failed to run on host 3cf99f574211.
240121-20:11:17,544 nipype.workflow ERROR:
	 Saving crash info to /neurodesktop-storage/somepatient/crash-20240121-201117-root-mrt_romeo_combine-9e4824f2-5b28-47b4-b8b0-f2e2b877dcf8.pklz
Traceback (most recent call last):
  File "/opt/miniconda-4.7.12.1/lib/python3.8/site-packages/nipype/pipeline/plugins/multiproc.py", line 67, in run_node
    result["result"] = node.run(updatehash=updatehash)
  File "/opt/miniconda-4.7.12.1/lib/python3.8/site-packages/nipype/pipeline/engine/nodes.py", line 527, in run
    result = self._run_interface(execute=True)
  File "/opt/miniconda-4.7.12.1/lib/python3.8/site-packages/nipype/pipeline/engine/nodes.py", line 645, in _run_interface
    return self._run_command(execute)
  File "/opt/miniconda-4.7.12.1/lib/python3.8/site-packages/nipype/pipeline/engine/nodes.py", line 771, in _run_command
    raise NodeExecutionError(msg)
nipype.pipeline.engine.nodes.NodeExecutionError: Exception raised while executing Node mrt_romeo_combine.

Cmdline:
	/opt/miniconda-4.7.12.1/lib/python3.8/site-packages/qsmxt/scripts/romeo_unwrapping.jl --no-rescale --compute-B0 B0.nii --phase /neurodesktop-storage/somepatient/result/workflow/sub-1/sub-1/qsmxt/mrt_romeo_combine/multi-echo-phase.nii --mag /neurodesktop-storage/somepatient/result/workflow/sub-1/sub-1/qsmxt/mrt_romeo_combine/multi-echo-mag.nii -t '[20.0 22.628 25.256 27.884]' --phase-offset-correction bipolar
Stdout:

Stderr:
	┌ Warning: Module WoodburyMatrices with build ID fafbfcfd-32e1-edab-0000-0669791b68c0 is missing from the cache.
	│ This may mean WoodburyMatrices [efce3f68-66dc-5838-9240-27a6d6f5f9b6] does not support precompilation but is imported by a module that does.
	└ @ Base loading.jl:1793
	┌ Warning: Module ChainRulesCore with build ID fafbfcfd-eb89-6df3-0000-066b237db0b1 is missing from the cache.
	│ This may mean ChainRulesCore [d360d2e6-b24c-11e9-a2a3-2a2ae2dbcce4] does not support precompilation but is imported by a module that does.
	└ @ Base loading.jl:1793
	┌ Error: Error during loading of extension ChainRulesCoreSparseArraysExt of ChainRulesCore, use `Base.retry_load_extensions()` to retry.
	│   exception =
	│    1-element ExceptionStack:
	│    Declaring __precompile__(false) is not allowed in files that are being precompiled.
	│    Stacktrace:
	│      [1] _require(pkg::Base.PkgId, env::Nothing)
	│        @ Base ./loading.jl:1825
	│      [2] _require_prelocked(uuidkey::Base.PkgId, env::Nothing)
	│        @ Base ./loading.jl:1660
	│      [3] _require_prelocked(uuidkey::Base.PkgId)
	│        @ Base ./loading.jl:1658
	│      [4] run_extension_callbacks(extid::Base.ExtensionId)
	│        @ Base ./loading.jl:1255
	│      [5] run_extension_callbacks(pkgid::Base.PkgId)
	│        @ Base ./loading.jl:1290
	│      [6] run_package_callbacks(modkey::Base.PkgId)
	│        @ Base ./loading.jl:1124
	│      [7] _require_prelocked(uuidkey::Base.PkgId, env::String)
	│        @ Base ./loading.jl:1667
	│      [8] macro expansion
	│        @ ./loading.jl:1648 [inlined]
	│      [9] macro expansion
	│        @ ./lock.jl:267 [inlined]
	│     [10] require(into::Module, mod::Symbol)
	│        @ Base ./loading.jl:1611
	│     [11] include
	│        @ ./Base.jl:457 [inlined]
	│     [12] include_package_for_output(pkg::Base.PkgId, input::String, depot_path::Vector{String}, dl_load_path::Vector{String}, load_path::Vector{String}, concrete_deps::Vector{Pair{Base.PkgId, UInt128}}, source::String)
	│        @ Base ./loading.jl:2049
	│     [13] top-level scope
	│        @ stdin:3
	│     [14] eval
	│        @ ./boot.jl:370 [inlined]
	│     [15] include_string(mapexpr::typeof(identity), mod::Module, code::String, filename::String)
	│        @ Base ./loading.jl:1903
	│     [16] include_string
	│        @ ./loading.jl:1913 [inlined]
	│     [17] exec_options(opts::Base.JLOptions)
	│        @ Base ./client.jl:305
	│     [18] _start()
	│        @ Base ./client.jl:522
	└ @ Base loading.jl:1261
	┌ Warning: The call to compilecache failed to create a usable precompiled cache file for Interpolations [a98d9a8b-a2ab-59e6-89dd-64a1c18fca59]
	│   exception = Required dependency ChainRulesCore [d360d2e6-b24c-11e9-a2a3-2a2ae2dbcce4] failed to load from a cache file.
	└ @ Base loading.jl:1818
	┌ Warning: The call to compilecache failed to create a usable precompiled cache file for LogExpFunctions [2ab3a3ac-af41-5b50-aa03-7779005ae688]
	│   exception = Required dependency DocStringExtensions [ffbed154-4ef7-542d-bbb7-c09d3a79fcae] failed to load from a cache file.
	└ @ Base loading.jl:1818
	┌ Warning: Module FixedPointNumbers with build ID fafbfcfd-7702-d143-0000-0673338fb613 is missing from the cache.
	│ This may mean FixedPointNumbers [53c48c17-4a7d-5ca2-90c5-79b7896eea93] does not support precompilation but is imported by a module that does.
	└ @ Base loading.jl:1793
	┌ Warning: The call to compilecache failed to create a usable precompiled cache file for MosaicViews [e94cdb99-869f-56ef-bcf0-1ae2bcbe0389]
	│   exception = Required dependency PaddedViews [5432bcbf-9aad-5242-b902-cca2824c8663] failed to load from a cache file.
	└ @ Base loading.jl:1818
	┌ Warning: The call to compilecache failed to create a usable precompiled cache file for HostCPUFeatures [3e5b6fbb-0976-4d2c-9146-d79de83f2fb0]
	│   exception = Required dependency BitTwiddlingConvenienceFunctions [62783981-4cbd-42fc-bca8-16325de8dc4b] failed to load from a cache file.
	└ @ Base loading.jl:1818
	┌ Warning: Module ImageCore with build ID ffffffff-ffff-ffff-0000-0677b624e64b is missing from the cache.
	│ This may mean ImageCore [a09fc81d-aa75-5fe9-8630-4744c3626534] does not support precompilation but is imported by a module that does.
	└ @ Base loading.jl:1793
	┌ Warning: Module ImageCore with build ID ffffffff-ffff-ffff-0000-0677b624e64b is missing from the cache.
	│ This may mean ImageCore [a09fc81d-aa75-5fe9-8630-4744c3626534] does not support precompilation but is imported by a module that does.
	└ @ Base loading.jl:1793
	┌ Warning: Module PolyesterWeave with build ID fafbfcfd-b1ce-e43e-0000-06810618da94 is missing from the cache.
	│ This may mean PolyesterWeave [1d0040c9-8b98-4ee7-8388-3f51789ca0ad] does not support precompilation but is imported by a module that does.
	└ @ Base loading.jl:1793
	┌ Warning: Module LoopVectorization with build ID ffffffff-ffff-ffff-0000-067d2cb40c22 is missing from the cache.
	│ This may mean LoopVectorization [bdcacae8-1622-11e9-2a5c-532679323890] does not support precompilation but is imported by a module that does.
	└ @ Base loading.jl:1793
	┌ Warning: Module ThreadingUtilities with build ID fafbfcfd-3c27-8a82-0000-0680aa41f8ef is missing from the cache.
	│ This may mean ThreadingUtilities [8290d209-cae3-49c0-8002-c8c24d57dab5] does not support precompilation but is imported by a module that does.
	└ @ Base loading.jl:1793
	┌ Warning: Module ThreadingUtilities with build ID fafbfcfd-3c27-8a82-0000-0680aa41f8ef is missing from the cache.
	│ This may mean ThreadingUtilities [8290d209-cae3-49c0-8002-c8c24d57dab5] does not support precompilation but is imported by a module that does.
	└ @ Base loading.jl:1793
	┌ Warning: Module MriResearchTools with build ID ffffffff-ffff-ffff-0000-066cdf521f71 is missing from the cache.
	│ This may mean MriResearchTools [557dad86-b9bd-4533-8525-1a39684d020f] does not support precompilation but is imported by a module that does.
	└ @ Base loading.jl:1793

	[279] signal (11.1): Segmentation fault

	[279] signal (11.1): Segmentation fault
	in expression starting at /opt/miniconda-4.7.12.1/lib/python3.8/site-packages/qsmxt/scripts/romeo_unwrapping.jl:3
	in expression starting at /opt/miniconda-4.7.12.1/lib/python3.8/site-packages/qsmxt/scripts/romeo_unwrapping.jl:3
	ijl_gc_safepoint at /opt/julia-1.9.3/bin/../lib/julia/libjulia-internal.so.1 (unknown line)
	ijl_task_get_next at /opt/julia-1.9.3/bin/../lib/julia/libjulia-internal.so.1 (unknown line)
	jl_gc_state_set at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/src/julia_threads.h:349 [inlined]
	maybe_collect at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/src/julia_threads.h:342 [inlined]
	jl_gc_pool_alloc_inner at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/src/gc.c:1450 [inlined]
	ijl_gc_pool_alloc at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/src/gc.c:1501
	poptask at ./task.jl:974
	unknown function (ip: 0x7fff91b04973)
	_jl_invoke at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/src/gf.c:2758 [inlined]
	ijl_apply_generic at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/src/gf.c:2940
	#gaussiansmooth3d!#24 at /opt/julia_depot/packages/MriResearchTools/59pXY/src/smoothing.jl:80
	unknown function (ip: 0x7fff89077c45)
	_jl_invoke at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/src/gf.c:2758 [inlined]
	ijl_apply_generic at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/src/gf.c:2940
	gaussiansmooth3d! at /opt/julia_depot/packages/MriResearchTools/59pXY/src/smoothing.jl:56
	#21 at ./threadingconstructs.jl:410
	unknown function (ip: 0x7fff89075b6f)
	_jl_invoke at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/src/gf.c:2758 [inlined]
	ijl_apply_generic at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/src/gf.c:2940
	jl_apply at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/src/julia.h:1880 [inlined]
	start_task at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/src/task.c:1092
	Allocations: 73048420 (Pool: 73013309; Big: 35111); GC: 110
	Segmentation fault
Traceback:
	Traceback (most recent call last):
	  File "/opt/miniconda-4.7.12.1/lib/python3.8/site-packages/nibabel/loadsave.py", line 42, in load
	    stat_result = os.stat(filename)
	FileNotFoundError: [Errno 2] No such file or directory: 'unwrapped.nii'

	During handling of the above exception, another exception occurred:

	Traceback (most recent call last):
	  File "/opt/miniconda-4.7.12.1/lib/python3.8/site-packages/nipype/interfaces/base/core.py", line 400, in run
	    outputs = self.aggregate_outputs(runtime)
	  File "/opt/miniconda-4.7.12.1/lib/python3.8/site-packages/nipype/interfaces/base/core.py", line 429, in aggregate_outputs
	    predicted_outputs = self._list_outputs()  # Predictions from _list_outputs
	  File "/opt/miniconda-4.7.12.1/lib/python3.8/site-packages/qsmxt/interfaces/nipype_interface_romeo.py", line 97, in _list_outputs
	    outputs['phase_unwrapped'] = split_multi_echo("unwrapped.nii", [extend_fname(f, "_romeo-unwrapped", ext="nii", out_dir=os.getcwd()) for f in self.inputs.phase])
	  File "/opt/miniconda-4.7.12.1/lib/python3.8/site-packages/qsmxt/interfaces/nipype_interface_romeo.py", line 20, in split_multi_echo
	    image4d = nib.load(in_path).get_fdata()
	  File "/opt/miniconda-4.7.12.1/lib/python3.8/site-packages/nibabel/loadsave.py", line 44, in load
	    raise FileNotFoundError(f"No such file or no access: '{filename}'")
	FileNotFoundError: No such file or no access: 'unwrapped.nii'


240121-20:11:17,560 nipype.workflow INFO:
	 [MultiProc] Running 0 tasks, and 0 jobs ready. Free memory (GB): 17.56/17.56, Free processors: 10/10.
240121-20:11:19,515 nipype.workflow INFO:
	 ***********************************
240121-20:11:19,520 nipype.workflow ERROR:
	 could not run node: workflow.sub-1.sub-1.qsmxt.mrt_romeo_combine
240121-20:11:19,523 nipype.workflow INFO:
	 crashfile: /neurodesktop-storage/somepatient/crash-20240121-201117-root-mrt_romeo_combine-9e4824f2-5b28-47b4-b8b0-f2e2b877dcf8.pklz
240121-20:11:19,528 nipype.workflow INFO:
	 ***********************************
Traceback (most recent call last):
  File "/opt/miniconda-4.7.12.1/bin/qsmxt", line 8, in <module>
    sys.exit(main())
  File "/opt/miniconda-4.7.12.1/lib/python3.8/site-packages/qsmxt/cli/main.py", line 1540, in main
    wf.run(
  File "/opt/miniconda-4.7.12.1/lib/python3.8/site-packages/nipype/pipeline/engine/workflows.py", line 638, in run
    runner.run(execgraph, updatehash=updatehash, config=self.config)
  File "/opt/miniconda-4.7.12.1/lib/python3.8/site-packages/nipype/pipeline/plugins/base.py", line 224, in run
    raise error from cause
RuntimeError: Traceback (most recent call last):
  File "/opt/miniconda-4.7.12.1/lib/python3.8/site-packages/nipype/pipeline/plugins/multiproc.py", line 67, in run_node
    result["result"] = node.run(updatehash=updatehash)
  File "/opt/miniconda-4.7.12.1/lib/python3.8/site-packages/nipype/pipeline/engine/nodes.py", line 527, in run
    result = self._run_interface(execute=True)
  File "/opt/miniconda-4.7.12.1/lib/python3.8/site-packages/nipype/pipeline/engine/nodes.py", line 645, in _run_interface
    return self._run_command(execute)
  File "/opt/miniconda-4.7.12.1/lib/python3.8/site-packages/nipype/pipeline/engine/nodes.py", line 771, in _run_command
    raise NodeExecutionError(msg)
nipype.pipeline.engine.nodes.NodeExecutionError: Exception raised while executing Node mrt_romeo_combine.

Cmdline:
	/opt/miniconda-4.7.12.1/lib/python3.8/site-packages/qsmxt/scripts/romeo_unwrapping.jl --no-rescale --compute-B0 B0.nii --phase /neurodesktop-storage/somepatient/result/workflow/sub-1/sub-1/qsmxt/mrt_romeo_combine/multi-echo-phase.nii --mag /neurodesktop-storage/somepatient/result/workflow/sub-1/sub-1/qsmxt/mrt_romeo_combine/multi-echo-mag.nii -t '[20.0 22.628 25.256 27.884]' --phase-offset-correction bipolar
Stdout:

Stderr:
	┌ Warning: Module WoodburyMatrices with build ID fafbfcfd-32e1-edab-0000-0669791b68c0 is missing from the cache.
	│ This may mean WoodburyMatrices [efce3f68-66dc-5838-9240-27a6d6f5f9b6] does not support precompilation but is imported by a module that does.
	└ @ Base loading.jl:1793
	┌ Warning: Module ChainRulesCore with build ID fafbfcfd-eb89-6df3-0000-066b237db0b1 is missing from the cache.
	│ This may mean ChainRulesCore [d360d2e6-b24c-11e9-a2a3-2a2ae2dbcce4] does not support precompilation but is imported by a module that does.
	└ @ Base loading.jl:1793
	┌ Error: Error during loading of extension ChainRulesCoreSparseArraysExt of ChainRulesCore, use `Base.retry_load_extensions()` to retry.
	│   exception =
	│    1-element ExceptionStack:
	│    Declaring __precompile__(false) is not allowed in files that are being precompiled.
	│    Stacktrace:
	│      [1] _require(pkg::Base.PkgId, env::Nothing)
	│        @ Base ./loading.jl:1825
	│      [2] _require_prelocked(uuidkey::Base.PkgId, env::Nothing)
	│        @ Base ./loading.jl:1660
	│      [3] _require_prelocked(uuidkey::Base.PkgId)
	│        @ Base ./loading.jl:1658
	│      [4] run_extension_callbacks(extid::Base.ExtensionId)
	│        @ Base ./loading.jl:1255
	│      [5] run_extension_callbacks(pkgid::Base.PkgId)
	│        @ Base ./loading.jl:1290
	│      [6] run_package_callbacks(modkey::Base.PkgId)
	│        @ Base ./loading.jl:1124
	│      [7] _require_prelocked(uuidkey::Base.PkgId, env::String)
	│        @ Base ./loading.jl:1667
	│      [8] macro expansion
	│        @ ./loading.jl:1648 [inlined]
	│      [9] macro expansion
	│        @ ./lock.jl:267 [inlined]
	│     [10] require(into::Module, mod::Symbol)
	│        @ Base ./loading.jl:1611
	│     [11] include
	│        @ ./Base.jl:457 [inlined]
	│     [12] include_package_for_output(pkg::Base.PkgId, input::String, depot_path::Vector{String}, dl_load_path::Vector{String}, load_path::Vector{String}, concrete_deps::Vector{Pair{Base.PkgId, UInt128}}, source::String)
	│        @ Base ./loading.jl:2049
	│     [13] top-level scope
	│        @ stdin:3
	│     [14] eval
	│        @ ./boot.jl:370 [inlined]
	│     [15] include_string(mapexpr::typeof(identity), mod::Module, code::String, filename::String)
	│        @ Base ./loading.jl:1903
	│     [16] include_string
	│        @ ./loading.jl:1913 [inlined]
	│     [17] exec_options(opts::Base.JLOptions)
	│        @ Base ./client.jl:305
	│     [18] _start()
	│        @ Base ./client.jl:522
	└ @ Base loading.jl:1261
	┌ Warning: The call to compilecache failed to create a usable precompiled cache file for Interpolations [a98d9a8b-a2ab-59e6-89dd-64a1c18fca59]
	│   exception = Required dependency ChainRulesCore [d360d2e6-b24c-11e9-a2a3-2a2ae2dbcce4] failed to load from a cache file.
	└ @ Base loading.jl:1818
	┌ Warning: The call to compilecache failed to create a usable precompiled cache file for LogExpFunctions [2ab3a3ac-af41-5b50-aa03-7779005ae688]
	│   exception = Required dependency DocStringExtensions [ffbed154-4ef7-542d-bbb7-c09d3a79fcae] failed to load from a cache file.
	└ @ Base loading.jl:1818
	┌ Warning: Module FixedPointNumbers with build ID fafbfcfd-7702-d143-0000-0673338fb613 is missing from the cache.
	│ This may mean FixedPointNumbers [53c48c17-4a7d-5ca2-90c5-79b7896eea93] does not support precompilation but is imported by a module that does.
	└ @ Base loading.jl:1793
	┌ Warning: The call to compilecache failed to create a usable precompiled cache file for MosaicViews [e94cdb99-869f-56ef-bcf0-1ae2bcbe0389]
	│   exception = Required dependency PaddedViews [5432bcbf-9aad-5242-b902-cca2824c8663] failed to load from a cache file.
	└ @ Base loading.jl:1818
	┌ Warning: The call to compilecache failed to create a usable precompiled cache file for HostCPUFeatures [3e5b6fbb-0976-4d2c-9146-d79de83f2fb0]
	│   exception = Required dependency BitTwiddlingConvenienceFunctions [62783981-4cbd-42fc-bca8-16325de8dc4b] failed to load from a cache file.
	└ @ Base loading.jl:1818
	┌ Warning: Module ImageCore with build ID ffffffff-ffff-ffff-0000-0677b624e64b is missing from the cache.
	│ This may mean ImageCore [a09fc81d-aa75-5fe9-8630-4744c3626534] does not support precompilation but is imported by a module that does.
	└ @ Base loading.jl:1793
	┌ Warning: Module ImageCore with build ID ffffffff-ffff-ffff-0000-0677b624e64b is missing from the cache.
	│ This may mean ImageCore [a09fc81d-aa75-5fe9-8630-4744c3626534] does not support precompilation but is imported by a module that does.
	└ @ Base loading.jl:1793
	┌ Warning: Module PolyesterWeave with build ID fafbfcfd-b1ce-e43e-0000-06810618da94 is missing from the cache.
	│ This may mean PolyesterWeave [1d0040c9-8b98-4ee7-8388-3f51789ca0ad] does not support precompilation but is imported by a module that does.
	└ @ Base loading.jl:1793
	┌ Warning: Module LoopVectorization with build ID ffffffff-ffff-ffff-0000-067d2cb40c22 is missing from the cache.
	│ This may mean LoopVectorization [bdcacae8-1622-11e9-2a5c-532679323890] does not support precompilation but is imported by a module that does.
	└ @ Base loading.jl:1793
	┌ Warning: Module ThreadingUtilities with build ID fafbfcfd-3c27-8a82-0000-0680aa41f8ef is missing from the cache.
	│ This may mean ThreadingUtilities [8290d209-cae3-49c0-8002-c8c24d57dab5] does not support precompilation but is imported by a module that does.
	└ @ Base loading.jl:1793
	┌ Warning: Module ThreadingUtilities with build ID fafbfcfd-3c27-8a82-0000-0680aa41f8ef is missing from the cache.
	│ This may mean ThreadingUtilities [8290d209-cae3-49c0-8002-c8c24d57dab5] does not support precompilation but is imported by a module that does.
	└ @ Base loading.jl:1793
	┌ Warning: Module MriResearchTools with build ID ffffffff-ffff-ffff-0000-066cdf521f71 is missing from the cache.
	│ This may mean MriResearchTools [557dad86-b9bd-4533-8525-1a39684d020f] does not support precompilation but is imported by a module that does.
	└ @ Base loading.jl:1793

	[279] signal (11.1): Segmentation fault

	[279] signal (11.1): Segmentation fault
	in expression starting at /opt/miniconda-4.7.12.1/lib/python3.8/site-packages/qsmxt/scripts/romeo_unwrapping.jl:3
	in expression starting at /opt/miniconda-4.7.12.1/lib/python3.8/site-packages/qsmxt/scripts/romeo_unwrapping.jl:3
	ijl_gc_safepoint at /opt/julia-1.9.3/bin/../lib/julia/libjulia-internal.so.1 (unknown line)
	ijl_task_get_next at /opt/julia-1.9.3/bin/../lib/julia/libjulia-internal.so.1 (unknown line)
	jl_gc_state_set at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/src/julia_threads.h:349 [inlined]
	maybe_collect at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/src/julia_threads.h:342 [inlined]
	jl_gc_pool_alloc_inner at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/src/gc.c:1450 [inlined]
	ijl_gc_pool_alloc at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/src/gc.c:1501
	poptask at ./task.jl:974
	unknown function (ip: 0x7fff91b04973)
	_jl_invoke at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/src/gf.c:2758 [inlined]
	ijl_apply_generic at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/src/gf.c:2940
	#gaussiansmooth3d!#24 at /opt/julia_depot/packages/MriResearchTools/59pXY/src/smoothing.jl:80
	unknown function (ip: 0x7fff89077c45)
	_jl_invoke at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/src/gf.c:2758 [inlined]
	ijl_apply_generic at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/src/gf.c:2940
	gaussiansmooth3d! at /opt/julia_depot/packages/MriResearchTools/59pXY/src/smoothing.jl:56
	#21 at ./threadingconstructs.jl:410
	unknown function (ip: 0x7fff89075b6f)
	_jl_invoke at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/src/gf.c:2758 [inlined]
	ijl_apply_generic at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/src/gf.c:2940
	jl_apply at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/src/julia.h:1880 [inlined]
	start_task at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/src/task.c:1092
	Allocations: 73048420 (Pool: 73013309; Big: 35111); GC: 110
	Segmentation fault
Traceback:
	Traceback (most recent call last):
	  File "/opt/miniconda-4.7.12.1/lib/python3.8/site-packages/nibabel/loadsave.py", line 42, in load
	    stat_result = os.stat(filename)
	FileNotFoundError: [Errno 2] No such file or directory: 'unwrapped.nii'

	During handling of the above exception, another exception occurred:

	Traceback (most recent call last):
	  File "/opt/miniconda-4.7.12.1/lib/python3.8/site-packages/nipype/interfaces/base/core.py", line 400, in run
	    outputs = self.aggregate_outputs(runtime)
	  File "/opt/miniconda-4.7.12.1/lib/python3.8/site-packages/nipype/interfaces/base/core.py", line 429, in aggregate_outputs
	    predicted_outputs = self._list_outputs()  # Predictions from _list_outputs
	  File "/opt/miniconda-4.7.12.1/lib/python3.8/site-packages/qsmxt/interfaces/nipype_interface_romeo.py", line 97, in _list_outputs
	    outputs['phase_unwrapped'] = split_multi_echo("unwrapped.nii", [extend_fname(f, "_romeo-unwrapped", ext="nii", out_dir=os.getcwd()) for f in self.inputs.phase])
	  File "/opt/miniconda-4.7.12.1/lib/python3.8/site-packages/qsmxt/interfaces/nipype_interface_romeo.py", line 20, in split_multi_echo
	    image4d = nib.load(in_path).get_fdata()
	  File "/opt/miniconda-4.7.12.1/lib/python3.8/site-packages/nibabel/loadsave.py", line 44, in load
	    raise FileNotFoundError(f"No such file or no access: '{filename}'")
	FileNotFoundError: No such file or no access: 'unwrapped.nii'

(base) root@3cf99f574211:/neurodesktop-storage/somepatient# qsmxt --version
[INFO]: QSMxT v6.3.2
```

</details>

<details>
  <summary>Click me for the nifti info</summary>

```
(base) anat % mrinfo *.nii.gz
************************************************
Image name:          "sub-1_echo-1_part-mag_MEGRE.nii.gz"
************************************************
  Dimensions:        512 x 512 x 140
  Voxel size:        0.4688 x 0.4688 x 1
  Data strides:      [ 1 2 3 ]
  Format:            NIfTI-1.1 (GZip compressed)
  Data type:         32 bit float (little endian)
  Intensity scaling: offset = 0, multiplier = 1
  Transform:               0.9949    -0.09926    -0.01871      -103.7
                          0.09527      0.9837     -0.1526      -97.75
                          0.03355        0.15      0.9881       -38.7
  comments:          TE=20;Time=113458.000
************************************************
Image name:          "sub-1_echo-1_part-phase_MEGRE.nii.gz"
************************************************
  Dimensions:        512 x 512 x 140
  Voxel size:        0.4688 x 0.4688 x 1
  Data strides:      [ 1 2 3 ]
  Format:            NIfTI-1.1 (GZip compressed)
  Data type:         32 bit float (little endian)
  Intensity scaling: offset = 0, multiplier = 1
  Transform:               0.9949    -0.09926    -0.01871      -103.7
                          0.09527      0.9837     -0.1526      -97.75
                          0.03355        0.15      0.9881       -38.7
  comments:          TE=20;Time=113458.000
************************************************
Image name:          "sub-1_echo-2_part-mag_MEGRE.nii.gz"
************************************************
  Dimensions:        512 x 512 x 140
  Voxel size:        0.4688 x 0.4688 x 1
  Data strides:      [ 1 2 3 ]
  Format:            NIfTI-1.1 (GZip compressed)
  Data type:         32 bit float (little endian)
  Intensity scaling: offset = 0, multiplier = 1
  Transform:               0.9949    -0.09926    -0.01871      -103.7
                          0.09527      0.9837     -0.1526      -97.75
                          0.03355        0.15      0.9881       -38.7
  comments:          TE=23;Time=113458.000
************************************************
Image name:          "sub-1_echo-2_part-phase_MEGRE.nii.gz"
************************************************
  Dimensions:        512 x 512 x 140
  Voxel size:        0.4688 x 0.4688 x 1
  Data strides:      [ 1 2 3 ]
  Format:            NIfTI-1.1 (GZip compressed)
  Data type:         32 bit float (little endian)
  Intensity scaling: offset = 0, multiplier = 1
  Transform:               0.9949    -0.09926    -0.01871      -103.7
                          0.09527      0.9837     -0.1526      -97.75
                          0.03355        0.15      0.9881       -38.7
  comments:          TE=23;Time=113458.000
************************************************
Image name:          "sub-1_echo-3_part-mag_MEGRE.nii.gz"
************************************************
  Dimensions:        512 x 512 x 140
  Voxel size:        0.4688 x 0.4688 x 1
  Data strides:      [ 1 2 3 ]
  Format:            NIfTI-1.1 (GZip compressed)
  Data type:         32 bit float (little endian)
  Intensity scaling: offset = 0, multiplier = 1
  Transform:               0.9949    -0.09926    -0.01871      -103.7
                          0.09527      0.9837     -0.1526      -97.75
                          0.03355        0.15      0.9881       -38.7
  comments:          TE=25;Time=113458.000
************************************************
Image name:          "sub-1_echo-3_part-phase_MEGRE.nii.gz"
************************************************
  Dimensions:        512 x 512 x 140
  Voxel size:        0.4688 x 0.4688 x 1
  Data strides:      [ 1 2 3 ]
  Format:            NIfTI-1.1 (GZip compressed)
  Data type:         32 bit float (little endian)
  Intensity scaling: offset = 0, multiplier = 1
  Transform:               0.9949    -0.09926    -0.01871      -103.7
                          0.09527      0.9837     -0.1526      -97.75
                          0.03355        0.15      0.9881       -38.7
  comments:          TE=25;Time=113458.000
************************************************
Image name:          "sub-1_echo-4_part-mag_MEGRE.nii.gz"
************************************************
  Dimensions:        512 x 512 x 140
  Voxel size:        0.4688 x 0.4688 x 1
  Data strides:      [ 1 2 3 ]
  Format:            NIfTI-1.1 (GZip compressed)
  Data type:         32 bit float (little endian)
  Intensity scaling: offset = 0, multiplier = 1
  Transform:               0.9949    -0.09926    -0.01871      -103.7
                          0.09527      0.9837     -0.1526      -97.75
                          0.03355        0.15      0.9881       -38.7
  comments:          TE=28;Time=113458.000
************************************************
Image name:          "sub-1_echo-4_part-phase_MEGRE.nii.gz"
************************************************
  Dimensions:        512 x 512 x 140
  Voxel size:        0.4688 x 0.4688 x 1
  Data strides:      [ 1 2 3 ]
  Format:            NIfTI-1.1 (GZip compressed)
  Data type:         32 bit float (little endian)
  Intensity scaling: offset = 0, multiplier = 1
  Transform:               0.9949    -0.09926    -0.01871      -103.7
                          0.09527      0.9837     -0.1526      -97.75
                          0.03355        0.15      0.9881       -38.7
  comments:          TE=28;Time=113458.000
```

</details>

<details>

<summary>Click me for an example json</summary>

```
{
	"Modality": "MR",
	"MagneticFieldStrength": 3,
	"ImagingFrequency": 127.7576,
	"Manufacturer": "GE",
	"ManufacturersModelName": "DISCOVERY MR750",
	"InstitutionName": "Hospital",
	"DeviceSerialNumber": "REDACTED",
	"StationName": "REDACTED",
	"BodyPartExamined": "BRAIN",
	"PatientPosition": "HFS",
	"ProcedureStepDescription": "REDACTED",
	"SoftwareVersions": "30\\LX\\SIGNA_LX1.MR30.0_R01_2242.a",
	"MRAcquisitionType": "3D",
	"SeriesDescription": "Brain Ax 3D  SWI",
	"ProtocolName": "! REDACTED",
	"ScanningSequence": "GR",
	"SequenceVariant": "SS\\SP\\SK",
	"ScanOptions": "FAST_GEMS\\FC\\ACC_GEMS\\PFP",
	"PulseSequenceName": "SWAN",
	"ImageType": ["ORIGINAL", "PRIMARY", "OTHER", "IMAGINARY"],
	"SeriesNumber": 10,
	"AcquisitionTime": "11:34:58.000000",
	"AcquisitionNumber": 1,
	"SliceThickness": 1,
	"SpacingBetweenSlices": 1,
	"SAR": 0.167528,
	"EchoTime": 0.02,
	"RepetitionTime": 0.0305,
	"SpoilingState": true,
	"FlipAngle": 15,
	"ShimSetting": [
		101,
		10,
		-294	],
	"PrescanReuseString": "TG/s9,CF/s2,AS/s2,RN/s1,Cal/s4",
	"CoilString": "8HRBRAIN",
	"PercentPhaseFOV": 80,
	"PercentSampling": 85.3711,
	"EchoTrainLength": 4,
	"AcquisitionMatrixPE": 240,
	"ReconMatrixPE": 512,
	"ParallelReductionFactorInPlane": 2,
	"PixelBandwidth": 488.281,
	"PhaseEncodingAxis": "i",
	"ImageOrientationPatientDICOM": [
		0.99489,
		0.09527,
		-0.03355,
		-0.09926,
		0.98369,
		-0.15001	],
	"InPlanePhaseEncodingDirectionDICOM": "ROW",
	"ConversionSoftware": "dcm2niix",
	"ConversionSoftwareVersion": "v1.0.20230411"
}
```

**Comment by astewartau** [comment]
Hi @LeeReid1, and thanks for reaching out! 

From what I can tell, your data is structured correctly, and it is encouraging that ROMEO has worked for you on similar data before. You're correct that the missing 'unwrapped.nii' file results from the ROMEO node failing for some reason. 

Unfortunately, I don't have a quick solution for this yet. I'm looking into possible causes, including whether switching the container to Julia v1.10 could fix this, or whether there is some issue with the precompilation for your architecture. I hope to reply soon with a new release or suggestions.

**Comment by LeeReid1** [comment]
Thanks @astewartau 
If you need anything verified on my end, please just let me know

**Comment by astewartau** [comment]
I've run QSMxT v6.3.2 on an ARM Macbook on our example dataset without this issue, so I think the problem may be related to the data or your environment. Although the environment is mostly containerised, the Home directory will be mounted to the container, and so the host ~/.julia directory will contain pre-compiled packages that may be leftover from your outside Julia version. I'd recommend deleting or renaming this folder to allow it to regenerate and see if it works. 

If that still doesn't solve the problem, we can verify whether it's a data problem by running the same test I did that passed as follows:

```
# install osfclient and download test data
pip install osfclient > /dev/null 2>&1
osf -p ru43c clone . > /dev/null 2>&1
tar xf osfstorage/dicoms-unsorted.tar -C .

# sort DICOMs
dicom-sort dicoms-unsorted dicoms-sorted

# convert DICOMs to  BIDS
dicom-convert dicoms-sorted bids --auto_yes --qsm_protocol_patterns '*qsm*' --t1w_protocol_patterns '*mp2rage*'

# run QSMxT
qsmxt bids qsm --premade fast --auto_yes --debug
```

If this test succeeds without issue, it would be interesting to learn more about your data. In this case, I could provide a link for you to upload it, and then I can try debugging it to see whether there is something about your data that is a problem for QSMxT.

If the test fails, then I'd recommend trying to process your data on [Neurodesk Play](https://play.neurodesk.org). You can drag-and-drop your data into the Desktop of this free online platform and find QSMxT v6.3.2 in the applications menu (you may need to press 'Update' first for this to populate fully).

Hopefully, these steps can get us closer to a solution!

**Comment by astewartau** [comment]
Hi @LeeReid1, I just wanted to follow-up to see whether those suggested solutions worked for you and whether there's anything else I can do to help.

**Comment by LeeReid1** [comment]
Hi,

I'm sorry for the delay. I'm split across a lot of tasks and we decided to run an alternative (philips) acquisition through the pipeline, but it took a while to obtain scan time. Seems to work quite well.

So you're right in that something was odd with the original (GE) acquisition. If I can find time I'll try to run another GE sequence through it in the next few weeks and let you know how it plays out.

Thanks for your help!

**Comment by LeeReid1** [comment]
Just a follow up - have run both another Philips image from another scanner and it failed with the same issue.

Other GE data I can't run from DICOM, because we've found the Phase values output by the scanner have proven problematic too many times and have resorted to acquiring real/imag pairs which I then convert to phase myself. These niftis will run without exception.
The only issue is all processed images are coming out with spots, for want of a better word. Still trying to figure that one out, but I'm assuming it's unrelated to the exception listed above.

**Comment by astewartau** [comment]
Closing due to inactivity — this was on v6, and packaging and unwrapping have changed a lot since (including the v9 rewrite). If GE/Philips data still fails on a current release, please open a new issue with the logs and we'll investigate. Thanks @LeeReid1!
